<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>BankGothic Md BT</family>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>90</width>
    <height>90</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>401</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="text">
     <string>LIBRARY TRACKER</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="addBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>99</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD BOOK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="borrowBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>149</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BankGothic Md BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BORROW BOOK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>199</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BankGothic Md BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RETURN BOOK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showAllBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>249</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BankGothic Md BT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SHOW ALL BOOKS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showIssuedBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>299</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BankGothic Md BT</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SHOW BORROWED BOOK</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>129</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="titleEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>179</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="authorEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>229</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="userEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>279</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>129</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BankGothic Md BT</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BOOK ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>179</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BankGothic Md BT</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Sunken</enum>
    </property>
    <property name="text">
     <string>TITLE:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>229</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BankGothic Md BT</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AUTHOR:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>279</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BankGothic Md BT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>USER NAME:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="outputArea">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>531</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BankGothic Md BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BOOK LIST:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
